scripts/sv/scanBam.sh

```diff
@@ -2,30 +2,33 @@
 
 set -eu
 
-if [[ -z ${GATK_DIR+x} || -z ${CLUSTER_NAME+x} || -z ${MASTER_NODE+x} || -z ${PROJECT_OUTPUT_DIR+x} ]]; then 
-    if [[ "$#" -ne 3 ]]; then
-        echo "Please provide: local directory of GATK build, cluster name, output dir on the cluster"
+# either have the required variables set in the running environment, or provide them 
+#  in command line
+if [[ -z ${GATK_DIR+x} || -z ${CLUSTER_NAME+x} || -z ${PROJECT_OUTPUT_DIR+x} || -z ${INPUT_BAM+x} || -z ${REF_FASTA+x} || -z ${SKIP_INTERVAL_LIST+x} ]]; then 
+    if [[ "$#" -ne 6 ]]; then
+        echo -e "Please provide:"
+        echo -e "  [1] local directory of GATK build (required)"
+        echo -e "  [2] cluster name (required)"
+        echo -e "  [3] output directory on the cluster (required)"
+        echo -e "  [4] absolute path to the BAM on the cluster (index assumed accompanying the bam) (required)"
+        echo -e "  [5] absolute path to the reference fasta on the cluster (required)"
+        echo -e "  [6] absolute path to the skip interval on the cluster (required)"
     exit 1
     fi
     GATK_DIR=$1
     CLUSTER_NAME=$2
-    OUTPUT_DIR=$3
+
     MASTER_NODE="hdfs://""$CLUSTER_NAME""-m:8020"
-    PROJECT_OUTPUT_DIR="$MASTER_NODE"/"$OUTPUT_DIR"
+    PROJECT_OUTPUT_DIR="$MASTER_NODE"/$3
+    INPUT_BAM="$MASTER_NODE"/$4
+    REF_FASTA="$MASTER_NODE"/$5
+    SKIP_INTERVAL_LIST="$MASTER_NODE"/$6
 fi
 
-REFERENCE_LOCATION="$MASTER_NODE"/reference/Homo_sapiens_assembly19.fasta
-INPUT_BAM="$MASTER_NODE"/data/NA12878_PCR-_30X.bam
-SKIP_INTERVAL_LIST="$MASTER_NODE"/reference/GRCh37.kill.intervals
-
-echo "Assuming reference: " "$REFERENCE_LOCATION"
-echo "Assuming input bam: " "$INPUT_BAM"
-echo "Assuming reference skip list: " "$SKIP_INTERVAL_LIST"
-
 cd "$GATK_DIR" 
 
 ./gatk-launch FindBreakpointEvidenceSpark \
-    -R "$REFERENCE_LOCATION" \
+    -R "$REF_FASTA" \
     -I "$INPUT_BAM" \
     -O "$PROJECT_OUTPUT_DIR"/fastq \
     --exclusionIntervals "$SKIP_INTERVAL_LIST" \
```

Comment:
Just call it a "kmer_kill_list" or something similar, it doesn't matter how it was generated as long as it's still appropriate to use on hg19, which we think it is.